graspologic/match/gmp.py

```diff
@@ -130,6 +142,14 @@ def __init__(
         else:
             msg = '"gmp" must be a boolean'
             raise TypeError(msg)
+        if isinstance(padding, str) and padding in {"adopted", "naive"}:
+            self.padding = padding
+        elif isinstance(padding, str):
+            msg = 'Invalid "padding" parameter string'
```

Comment:
how did black not complain about using a ' instead of a "

Reply:
are you a wizard

Reply:
maybe different ticks? I used " ' " not " ` ".

Reply:
the exception to blacks rule is strings that have quotes within them like the one here. see https://black.readthedocs.io/en/stable/the_black_code_style.html#strings
though Ali is, in fact, a wizard.

Reply:
(cause " ... "padding" ..." would break)